research/sentiment_analysis/README.md

```diff
@@ -0,0 +1,23 @@
+# Sentiment Analysis
+## Overview
+This is an implementation of the Sentiment Analysis model as described in the [this](https://arxiv.org/abs/1412.1058) paper. The implementation is with the reference to [paddle version](https://github.com/mlperf/reference/tree/master/sentiment_analysis/paddle).
```

Comment:
nit: the [paper](https://arxiv.org/abs/1412.1058)

Reply:
ack